app/utils/application-state-utils.ts

```diff
@@ -598,9 +599,16 @@ async function getAppealApplicationNextStep(req: Request) {
         };
       } else if (ftpaEnabled && APPLICANT_TYPE.RESPONDENT === ftpaApplicantType) {
         const ftpaDecision = req.session.appeal.ftpaRespondentDecisionOutcomeType || req.session.appeal.ftpaRespondentRjDecisionOutcomeType;
-        doThisNextSection = {
-          descriptionParagraphs: i18n.pages.overviewPage.doThisNext.ftpaDecided.respondent[ftpaDecision]
-        };
+        if (ftpaSetAsideFeatureEnabled) {
+          doThisNextSection = {
+            cta: {},
```

Comment:
@Ryan-Nesbitt , can you do a quick check with respondent FTPA 'Granted' (Just simply redo the decideFtpa event on same case). Wouldn't it have 'Do this next' for them, we should keep them as 'Nothing to do next' even though toggle is on right?

Reply:
@ThomasKKC I checked and you're right. Added a ftpaDecision check for 'reheardRule35' as well as the toggle being on for the 'Do This Next' to show, otherwise show 'Nothing to do next'